samples/net/lwm2m_client/prj.conf

```diff
@@ -29,6 +29,13 @@ CONFIG_LWM2M_IPSO_SUPPORT=y
 CONFIG_LWM2M_IPSO_TEMP_SENSOR=y
```

Comment:
Not necessarily part of this PR, but I just noticed that we are missing lwm2m settings from `tests/net/all/prj.conf` file. The file tries to enable max number of options from networking subsystem. It would be great to have lwm2m there too. There is `tests/net/all/check_net_options.sh` that is currently missing lwm2m checks and can be used to generate a list of currently missing options from prj.conf file. As I said, no need to add this support in this PR (so that we get this one merged sooner than later) but it would be nice to have lwm2m in tests/net/all at some point.

Reply:
Referencing this issue so we can track this (which has been open for some time): https://github.com/zephyrproject-rtos/zephyr/issues/4634